workers/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/gocraft/work"
+	"github.com/gomodule/redigo/redis"
+	"github.com/gutakk/go-google-scraper/config"
+	"github.com/gutakk/go-google-scraper/db"
+	"github.com/gutakk/go-google-scraper/workers/jobs"
+)
+
+func init() {
+	config.LoadEnv()
+	_ = db.ConnectDB()
+}
+
+// Make a redis pool
+var redisPool = &redis.Pool{
+	MaxActive: 5,
+	MaxIdle:   5,
+	Wait:      true,
+	Dial: func() (redis.Conn, error) {
+		return redis.Dial("tcp", "localhost:6379")
+	},
+}
+
+func main() {
```

Comment:
Would this restart all pending jobs (not started and below `jobs.maxFails`) when the worker process is stopped and restarted?

Reply:
Yes. here are the steps as following GIF
1. Start app
2. Uploads keywords and then app will enqueue all the job
3. Start worker
4. Worker executes pending jobs

![lrUEs3Y3WJ](https://user-images.githubusercontent.com/29707647/102449101-22d29500-4066-11eb-96bc-b30570618645.gif)

Reply:
Then this solution seems very complete 🤗